source/auth_n.rst

```diff
@@ -5137,10 +5076,10 @@ If authentication is successful, you will get back the Account:
     .. literalinclude:: code/python/authentication/mfa_auth_account_resp.py
         :language: python
 
-Next, you will need to retrieve the Account's ``factors`` collection:
-
 .. only:: rest
 
+Next, you will need to retrieve the Account's ``factors`` collection:
```

Comment:
This is duplicated in Java and in REST: both REST and Java codes are shown in both guides